docs/inputs.rst

```diff
@@ -73,7 +73,7 @@ An example of an orbit file in cometary format, with each object ID represented
 +-------------+----------------------------------------------------------------------------------+
 
 **Header line**
-The first row in the orbit file must be a header started with ‘!!’ to denote it as the header row::
+The first row in the orbit file must be a header started with "!!" to denote it as the header row::
```

Comment:
This isn't true! The first row needs to start with a column headed `ObjID`, no exclamation marks. This is a holdover from a very old version of the code.

Reply:
@steph. This is not complete. Let’s not go through every change. Required. It’s definitely not complete.

Reply:
That's fine - if you check my overall review, I've approved the PR and noted that all the things I've commented on can easily be addressed much later.